### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by RWUBAKWANAYO**
##In this project we:

- Add work section and about section in HTML file
- Add styles to created section
- setup mobile version of new sections
- Fix linter errors in the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,19 +24,200 @@
...
+        <h3>LET'S CONTACT</h3>
+        <span class="about-media-link-span">
+						<a href="https://github.com/" target="”_blank”" class="media-icon about-media-icon"
+							><i class="fa-brands fa-github"></i
+						></a>
+						<a href="https://www.linkedin.com/" target="”_blank”" class="media-icon about-media-icon"
+							><i class="fa-brands fa-linkedin-in"></i
+						></a>
+						<a href="https://angel.co/" target="”_blank”" class="media-icon about-media-icon"
+							><i class="fa-brands fa-angellist"></i
```
Comment (RolandM99) [inline index.html]: - [ ] Please, Kindly wrap your list of social media icons with a `<ul>` tag and each element inside with a `<li>` tag. 
- **Example:**
```
<ul>
     <li><a>....</a></li>
     <li><a>....</a></li>
     <li><a>....</a></li>
</ul>

File: index.html [hunk 2]
```diff
@@ -24,19 +24,200 @@
...
+      <li>
+        <a href="https://angel.co/" target="”_blank”" class="media-icon"><i class="fa-brands fa-angellist"></i
+					></a>
+      </li>
     </ul>
   </section>
+  <section class="work-container">
+    <div class="work-wrapper">
+      <div class="work-sub-cont work-img-wrapper">
+        <img src="./images/tonic.jpg" alt="work pictictre" />
```
Comment (RolandM99) [inline index.html:58]: - [ ] Make sure that you follow correctly Figma design for your first work card.
### Figma design: 
image
### Your design: 
image
- Please, kindly fix this issue by fixing the image above your content.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @RWUBAKWANAYO , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -152,6 +164,253 @@ header {
...
+  border-radius: 8px;
+  border: 1px var(--lightest-blue) solid;
+  font-family: var(--poppins);
+  font-size: calc(var(--extra-small) - 1px);
+  font-weight: 500;
+  background-color: var(--white-color);
+  color: var(--lightest-blue);
+  cursor: pointer;
+  transition: 0.3s ease-in-out;
+}
```
Comment (vic778) [inline style.css:280]: - [OPTIONAL]  kindly make sure you're using the same font style and font-weight as the [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png) template

**Review by vic778 — APPROVED** [review]
Hi @RWUBAKWANAYO  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RWUBAKWANAYO** [comment]
Merge changes to the main branch  after the pull request approved

**Pull request merged**